Dockerfile

```diff
@@ -25,63 +25,70 @@
 # Use CentOS7:
 FROM centos:7
 
-# Configure virtualenvwrapper root directory
-ENV WORKON_HOME=/usr/local/var
-ENV PROJECT_HOME=/usr/local/var
-# Prepare provisioning script:
-COPY scripts/provision-web.sh /tmp/
-
-# Install CERN Open Data Portal web node pre-requisites:
-RUN /tmp/provision-web.sh
+# Configuration for CERN Open Data Portal instance:
+ENV APP_INSTANCE_PATH=/usr/local/var/cernopendata/var/cernopendata-instance
```

Comment:
this might not be the best path ever, but changing it now would involve more changes on the running instances+testing just in case